modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/sql/IgniteSqlUpdate.java

```diff
@@ -37,7 +38,7 @@
 public class IgniteSqlUpdate extends SqlUpdate {
 
     /** UPDATE operator. */
-    protected static final class Operator extends IgniteSqlSpecialOperator {
+    protected static final class Operator extends SqlSpecialOperator {
```

Comment:
This was a typo. UPDATE is not a special operator at all.

Reply:
the comment is funny, due to it is still Special operator